src/components/input/DurationInput.js

```diff
@@ -11,19 +11,38 @@ const DurationInput = ({ value, onChange, onKeyDown }) => {
   };
 
   const handleMinutesChange = (_, minutes) => {
-    onChange &&
+    if (!onChange) {
+      return;
+    }
+
+    if (minutes >= 60) {
       onChange({
-        ...value,
-        minutes,
+        hours: (value?.hours || 0) + 1,
+        minutes: 0,
+      });
+      return;
+    }
+
+    if (minutes < 0) {
+      onChange({
+        hours: (value?.hours || 1) - 1,
```

Comment:
Should it also make sure to not output negative hours here?

Reply:
It will actually take it into account, if `value.hours = 0`, it will fall back to 1 🙈 Bit weird to read though so I understand the confusion

Reply:
right I see 😅